bim2sim/sim_settings.py

```diff
@@ -261,39 +396,27 @@ def update_from_config(self, config):
                             f'Please use strings only in config.')
         logger.info(f'Loaded {n_loaded_settings} settings from config file.')
 
-    dymola_simulation = Setting(
+    dymola_simulation = BooleanSetting(
         default=False,
-        choices={
-            True: 'Run a Simulation with Dymola afterwards',
-            False: 'Run no Simulation and only export model'
-        },
-        description='Run a Simulation after model export?',
+        description='Run a Simulation with Dymola after model export?',
         for_frontend=True
     )
-    create_external_elements = Setting(
+    create_external_elements = BooleanSetting(
         default=False,
-        choices={
-            True: 'Create external elements',
-            False: 'Create external elements'
-        },
         description='Create external elements?',
         for_frontend=True
     )
-    max_wall_thickness = Setting(
+    max_wall_thickness = NumberSetting(
         default=0.3,
-        choices={
-            1e-3: 'Tolerance only for opening displacement',
-            0.30: 'Maximum Wall Thickness of 0.3m',
-            0.35: 'Maximum Wall Thickness of 0.35m',
-            0.40: 'Maximum Wall Thickness of 0.4m'
-        },
+        max_value=0.40,
```

Comment:
I would suggest setting the maximum value to at least 0.5m, rather 0.6m (for very old buildings this may be required).

Reply:
solved